src/app/containers/Include/index.amp.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { string } from 'prop-types';
+import styled from 'styled-components';
+import { AmpImg } from '@bbc/psammead-image';
+import { GridItemConstrainedMedium } from '#lib/styledGrid';
+
+const getSrc = href => {
+  const path = href.split('/').slice(3).join('/');
```

Comment:
```suggestion
  const path = href.replace(/https:\/\/smallprox.vj(.*).tools.bbc.co.uk\//, '');
```
I think it might be a bit clearer if we removed the smallprox part of the URL using a regex. Just a bit easier to picture what we are replacing rather than using relying on indexes from a split.

Could we also raise the fact the smallprox domain is in the response with Ares? Ideally this should not be in the data as it exposes an internal endpoint; that would then mean we could remove this code too.

Reply:
> I think it might be a bit clearer if we removed the smallprox part of the URL using a regex. Just a bit easier to picture what we are replacing rather than using relying on indexes from a split.

@andrewscfc this would mean using the smallprox url in the test as mock data 

> Could we also raise the fact the smallprox domain is in the response with Ares? Ideally this should not be in the data as it exposes an internal endpoint; that would then mean we could remove this code too.

I definitely agree with this

Reply:
>  this would mean using the smallprox url in the test as mock data

I didn't think about that, I don't see that as being an issue personally, if it's already in the fixture data it is already visible. Did you get a chance to raise this with ares, we could potentially get this sorted before this PR is merged?